nipype/algorithms/confounds.py

```diff
@@ -301,23 +301,29 @@ def _list_outputs(self):
 
 class CompCorInputSpec(BaseInterfaceInputSpec):
     realigned_file = File(exists=True, mandatory=True,
-                          desc='already realigned brain image (4D)')
-    mask_file = File(exists=True, desc='mask file that determines ROI (3D)')
-    components_file = File('components_file.txt', exists=False,
-                           usedefault=True,
-                           desc='filename to store physiological components')
+        desc='already realigned brain image (4D)')
+    mask_files = InputMultiPath(File(exists=True,
```

Comment:
this should go through a deprecation cycle. so keep the old name and point to the new name. 

e.g.: https://github.com/nipy/nipype/blob/master/nipype/interfaces/spm/preprocess.py#L812